### DISCUSSION TITLE
Iron Idol changes

### DISCUSSION THREAD

**Discussion by Xeko-92** [sunflower-land/sunflower-land#2272] [Ideas]
Before anyone mints the iron idol, can we please discuss the idea of changing it to be either 0.5 iron and a 400 supply or even 0.25 iron and 800 supply. It's already far better than any of the moles and you know a bunch of people are just going to get shit RNG and be at 39/40 starfish with nothing they can do and be pissed off when they don't get this. It's such a powerful NFT for no real reason. It's not some super expensive NFT that only whales can get, it's not like the fruit NFTs that require dedication and being consistent (other than just digging daily), it all comes down to RNG. You can't even increase your odds with a drill since the 40 starfish are common and you can't get them with drills.

Obviously some people will still miss out on the idol no matter what, but if we increase the supply and lower the bonus, more people will get it and it will be cheaper to buy on the secondary market if you happen to not get it in time. Missing out on +1 iron forever (without spending $500+) because your RNG was slightly worse than someone else's is going to feel like absolute shit.

This is a worse minting system than the auction that's trying to be changed. There have been a bunch of people in discord saying that they sold their star fish after getting the treasure map because that's the point of the bounties. Now those people are entirely out of the race.

For reference, I have 10 starfish so I'm still in the race for top 200. I still think this is terrible way to distribute such a powerful NFT.

**Comment by travlv** [comment]
I agree! That is insane to release such powerful and highly expected nft to be so limited and to require such random unrelated ingredients! Like what's going on, maybe just add god mode to yourself and your friends, it will be the same! Freaking starfish and now those who just hasn't been using the mechanic of the game that you were actully meant to use such as selling bounties   - they prospect!
Me and a lot of other people were expecting iron nft and were saving iron but bam, idk, devs tripped on lsd or something and decided it's gonna require starfish.. 
and another thing is +1. Are you kidding me?? What about trying not to separate too much players and to give some players too much advance?? 
It is all looks so weird, idk.
My farm is 133341, bumpking 1374. I'm not an OG, but not a newbie for sure. Been playing long time, put decent amount of money in, haven't got anything out yet. Been happy with the game and devs before but this feels like unexpected male genitalia to my face!

**Reply by usernrone** [reply]
Why is starfish so wrong item to collect it's the same as compasses?.. No i don't think the items is the problem it's yours feeling like being left out and the story would be different if they who say it's wrong had a lot more starfish to start with than they got...
My thoughts about it maybe I'm wrong but often is it where it comes from...

**Reply by Xeko-92** [reply]
3 things.

1. The compasses were known to be needed immediately from the start.
2. Compasses can be obtained with drills so you have a way to "target" farm them.
3. The Heart of Davey Jones is a MUCH worse NFT than the Iron Idol, so if you miss out, it will probably be like a $10 NFT on opensea.

**Reply by travlv** [reply]
compasses are for the something that is TI related. TI is only. part of the game and some players may decide not. even participate in it at all. but idol is such essential and powerful nft.

**Reply by rasheed19871** [reply]
I totally agree in my opinion the boost should be lower and the supply shall increase because every one want it I am not saying 1000 pieces. But  200 com on guys what is it.

**Comment by usernrone** [comment]
I don't really have any problem or thoughts about it and don't see a need for an change, some items will be rarer than others.
I believe there will come others with boost for iron that will be obtanible by a larger group..

Iron idol is one of those item that will be super rare and it creates a buzz around it which will be recognized easier and create intresset on opensea and it attracts whales for even if whales takes a lot it gives so much back to everyone else and whales will be needed for a more stable econamy as long it's controlled...

I understand your thoughts about it but i don't see it as an negative thing and i will be happy for those how gets it..

**Reply by travlv** [reply]
it is one step from the situation when there is a new nft that gives you +5 gold and requires 5000 potatos for example. weird and unrelated.

**Reply by rasheed19871** [reply]
If +1 iron happen so can +5 gold, the whale will come maybe, but what about existing player

**Comment by JustJayHSP** [comment]
I'm fine with the Starfish and the Pearl, since that could make use of more Sand Shovelling in Treasure Island. Making that island more Active than it is right now or stay active longer until 2 and a half months has passed and a new set of rewards pop up.

Although I wont lie, I do want more supply towards Iron Idol and if nerfing its boost would provide better Supply that would be great.

Also since its an Iron Idol wouldn't it be better to ask for 50 - 100 Iron instead of 10 Gold?

We're around 15,000 Active Farm players and I think having more supply would benefit the people playing Sunflower Isles and retain them rather than a 200 Supply where everyone is desperately trying to get it, like its the savior of Iron gains.

Also there's a clear shortage of Stone rather than Iron right now. Would love it more if this was a stone boost. Golem+Mole for the less fortunate Island does not gather enough stone to keep mining Iron or Gold. So if we can just alter this to a Stone Idol instead that would be better in my book.

**Reply by travlv** [reply]
thank you for your opinion. I actually think it should require iron and quite a lot. maybe even 300-500

**Comment by Rook982** [comment]
A part of the reason they used TI components and not existing resources is because it gives everyone a fair shot to mint it even if it is RNG, I'm assuming. 

Now imagine it will need 500 Iron and 50 Gold to mint and still at 200 supply, It will be minted out so fast by high level farmers it will turn out to be another auction debacle.

Now with this mint requirements the newer players will be complaining again that they missed out on an NFT "since only older players can get it", "why invite new players if only old players can mint", etc. etc. same old same old. So you can't make everyone happy. People who really want it will still buy even at a high price point, Happened for QBKO, FB, Xmas Tree, GA, Parsnip, Wings.

I agree with lowering requirements and lowering the buff to increase supply, but making the requirements to only existing resources stone to gold, then the conflict will just shift to new players vs old players and "minting system bad". At least now everyone is united against bad RNG. Something else that could have been done better is transparency, if we knew or at least teased beforehand of upcoming drops then we could have prepared instead of sold TI items. They don't need to tell the exact item and number just that TI items could be used to craft this "X" NFT. Would have made people second guess selling their bounties.

**Reply by travlv** [reply]
disagree because TI is meant to be as a source of SFL. people. spend sfl by chopping wood and stone and then buying shovels. for a chance to get more sfl or something else. but with this approach still winning only those who don't need to sell bounties

**Reply by JustJayHSP** [reply]
This is why Im fine with the Starfish and Pearl requisite. It makes everyone who's actively shovelling have a chance at minting it. Although people who have used the Drill and had already minted Davy Jones will have a domino effect of getting StarFish drops more often than a normal farmer who shovels 30 a day only.

Still fair in my opionion to have Pirate Resources instead of the usual Materials that a normal Whale would always have.

**Reply by travlv** [reply]
in this exact case won those who didn't need to sell bounties because they had a lot of sfl

**Reply by JustJayHSP** [reply]
Travlv and now we know to not sell any StarFish and keep at least 1 pearl if you want to get an Iron Idol. We can probably sell everything else but who knows maybe the Devs will surprise us again with another Pirate NFT. 

Another panic moment right there, if that ever happens.

**Reply by Rook982** [reply]
TI is first and foremost a resource sink, not SFL source for players, end of the day SFL needs to keep an advantage of money going in and not money going out. I'm really sorry for those that sold and I agree that it could have been communicated better.

**Reply by drumdrumdrum3337** [reply]
Actually TI is not meant to be a "source of SFL", it is meant to be an SFL sink, meaning that it is designed to burn sfl, not to generate it, it is in the docs.

**Reply by travlv** [reply]
this whole game is meant as a $ sink, let's start from here. but TI is at least presented as one of sources of sfl

**Reply by Rook982** [reply]
Yes, but with low odds to make the resources you spent for shovels back to breakeven. Its a mechanic that allows you to put resources from main island to another form to get use. But unless you have the Luck of the Devil ain't no way you are making a profit with what you have already spent on TI. Better to have stuck with just selling your wood, stone or gold if you want to make money.

**Reply by Aminbtdds** [reply]
Imho, TI designed to be 20-30% basic resource sink overall, it was never meant to be new way to earn sfl.

**Comment by pappajan69** [comment]
i whit every one think there need to be 800 of them that give 0.25 boost we all need more stone

**Comment by drumdrumdrum3337** [comment]
I, for one, like the current mint and wouldn't go for depleting the boost. Here's why:

1. Allowing 400 or 800 players to mint the nft won't change the fact there'll always be a player who comes 401st or 801st and not able to mint it. Look at the recent Wendy mint - discord was was full of upset people for days because they didn't get to mint, and we are talking a +0.2 wood boost here. Players just have to accept the fact that this is a game, and they won't be able to get everything no matter how hard they try. You can have a perfect game with triple-double yet your team loses bcs opponent just scores from every silly situation. You do your best, and the rest is up to heavens, that's just how it is. 

2. I like that the nft is not expensive and iron related. This way, a complete newbie with just enough expansions to reach the TI can have a go at one of the most valuable nfts out there, and not just a small club of VIP folks. Don't get me wrong, I am one of them, but that club should not get everything, otherwise the game will suffocate itself.

Note: the only thing that I'd change is probably that starfish selection, to maybe something new that no one has, but this is really up to devs to decide. Maybe they want to appreciate hodlers more this way, and it is their decision to make.

**Reply by pappajan69** [reply]
this is one of the best game out there alot of poeple going to join the game yes new players must get chance at new items we want more poeple to join game and so far the OG PLAYERS have must more luck then the new players and they stock up so they dont need much as new player sell every thing they get so need to start from noting and try to get every thing to mint new nfts just 800 poeple geting some boost is way better then 200 it give more poeple chance to get somting

**Reply by travlv** [reply]
I just got used to when nfts boosting something specific require this exact resource. it was always like this before. maybe it needs to be changed and if i don't like it i should just sell my assets and go. but guess what, devs don't let you easily do that..

**Reply by Rook982** [reply]
Echo drum^3's post.

A lot of people will just get salty for any reason, Iron Idol's requirement is beginner friendly and makes sure that not only whales get it (but a majority will still do due to more chances to dig). Making it a race makes it a fair mint while they work on the improved minting system. 

Communication and transparency just needs to be better from the team.

**Reply by drumdrumdrum3337** [reply]
Devs were very open about the fact that there'll be different types of nfts - with small boosts (potato guy, wendy, tiki, etc), with bigger boost (lady bug), and big boosts (iron one); and also different ways to obtain them - via pure grind (bearry bear), during events (lunar calendar, maneki neko, etc) and some luck based ones (iron one). Having this diversity is just part of the fun, there's no reason why things should be always the same.

**Reply by Xeko-92** [reply]
Comparing this mint to the Wendy mint just proves my point even further. It was minted in the auction format that the devs are changing because everyone hates it.

As I said in my original post, yes, increasing the supply will still cause people to miss out, but it's one thing to miss out on a $100 NFT, it's another thing to miss out on $1000 NFT because of bad RNG.

Curious to know how many star fish you currently own.

**Reply by Rook982** [reply]
I have 5 starfishes and no pearls, I am all for a nerf in the efficiency of the idol to increase supply so it decreases in value, but as stated in a post above the variance of NFT acquisition types is a good thing to let them experiment and help them know whats right and wrong. So far its indeed undesirable to have an NFT based on luck but considering the web3 space RNG is a major entity so you can't fault that. They can still make future NFTs for Iron efficiency on those acquisition systems that are favorable to all.

**Reply by Xeko-92** [reply]
I also don't entirely have an issue with the way it's distributed, my issue is with the power of this NFT AND the way it's distributed. If it's going to be this powerful, it either needs to be a whale item that costs a lot to mint or it needs to be some sort of skill based mint or require a lot of effort like the fruit NFTs.

I think a lot of the people supporting the current item are in the top 100 of starfish.

**Reply by drumdrumdrum3337** [reply]
I have 8 starfish, and have no idea where it puts me in terms of chances to get to mint the item (I am quite sure I'd mint if it would be resource-heavy mint though). But, that's not the point, and you won't believe me, but I am completely fine if I am not going to mint it. That's a game, and as in every game I can lose, my luck can have a day off, it saved me in my life too many times for me to complain. 

What I am really interested in is what will become of this game, whether it'll develop the necessary model to live for years. One nft doesn't bother me at all.

**Reply by Xeko-92** [reply]
No, I do believe you. Some people don't really care about this kind of stuff, but others care a lot. I just think this isn't the best choice the devs could be making right now.

**Reply by usernrone** [reply]
No there is no differens to to miss out on those 2 and i don't believe there this is a bad way to mint out a high value (in-game) nft...
I can not see why it is so bad to miss out on an nft that is availble to all that have access to TI and thats great.
When it talks about value on second market it becomes more than if this nft is bad for the game or if it couse issue that will have an so negative effect on the game it brakes or anything, whats happening on second market is pure speculation and i don't think devs should think about that, that is players to decide and control...

**Comment by travlv** [comment]
so the situation is - they create rules and concept, you like it and get involved personally and financially, then they change it and you have no choice

**Reply by travlv** [reply]
iliterally. just 2-3 days ago had all the bounties and thought - i need sfl, should i sell them or get sfl some other way? 
but thought - bounties are definitely meant to be sold and can't have any unexpected utility.
and today bum! guess what..

**Reply by Rook982** [reply]
Compass was listed as a "bounty" and used for NFT and blacksmith NFT.

**Reply by travlv** [reply]
it was mentioned that it will have utility

**Reply by Rook982** [reply]
Then there was a precedent that existed.

precedent; 
    an earlier event or action that is regarded as an example or guide to be considered in subsequent similar circumstances

**Reply by travlv** [reply]
i guarantee that never before such powerfull nft was offered for such random weird price

**Reply by Rook982** [reply]
Then you'd rather prefer the Iron Idol to be another Wendy Mint debacle? With most owners only being whales? Also not random, some ancient Idol's are excavated in burial sites. Remains on theme for a Treasure Island theme NFT.

**Reply by travlv** [reply]
Wendy was for whales only??? i'm not a whale and I minted it easily. And if I'm a whale then I should be able to mint idol but i don't =)

**Reply by Rook982** [reply]
Post was incomplete so I'll lay it out here. You forgot your initial argument to make NFT requirements relevant to resources they buff in proportion to efficiency. With this it would make sense that it will need a lot of stone/iron/gold. With only 200 supply and a lot of people who have hoarded said resources it will recreate the problem of the Wendy mint + mostly Whale players who have qualified for having said resources. Starting another issue with new players and bad mint system.

My take on it and said all I need. You're free to take whatever interpretation you want.

**Comment by IvanSkazkin** [comment]
Personally, I don't like RNG. SFL has always liked that anyone with perseverance could get the coveted nft. It would be better in my opinion. Add a certain resource for example from 11 rafts. By extracting it, one could save up for nft. The resource could be seasonal. Then people would be stimulated to bumpkins their redneck more. And higher levels could start racing much earlier.

I have 0 pearls, but I spent about 50 gold. It's a terrible feeling of emptiness.

I'm level 40. They give me cakes. Why do I need them? I have to race for nft. I get cakes that I don't need at all. When will level 40+ be available again?

**Reply by travlv** [reply]
i spent around 100 gold and had 1 pearl but sold it. all i'm saying is why for black bearries you need blueberries? and. for monkey orange? there. is. some logical concept

**Comment by travlv** [comment]
new idea - 13.5 crabs + 1 pirate bounty +12 war bonds = god mode!

**Reply by usernrone** [reply]
No it's not then it's just for old players, all coming nft should be for all new or old players, not for a group where others being left just becouse they did't play "6 month" back... All new ntf will be harder for old players to get and that's great, usually it's always old players or whales that gets most of all but now it's equal chance for a new player to get one.. 
Chose starfish and pearl is the probably the best way to come as close we can to br fair for all..
There will alway be they that have a smaller chance to get it but we can't get rid of all  so it's perfect since there most be some gameplay behind it with some difficulties to get that will be harder for some.. 
It can not straght be an raffle then and it wouldn't bring much of any gameplay..
Everything doesn't have to have logic behind it and it's great no one can really anticipate and prepare for it and i love it..

**Comment by Xeko-92** [comment]
Can we get a comment on this from the devs please? @adamhannigan @Rofrtd @c-py 

I'm someone who's been playing this game since the beginning of SFF, and launching an NFT that's this powerful in this way is one of the worst decisions I've seen made since I've been here.

![image](https://user-images.githubusercontent.com/123136614/220224728-edb23aea-08a0-4b2b-82cf-9c694ad3fadf.png)

**Comment by SarielWang93** [comment]
I think the use of starfish and pearls as materials at this moment is deliberate and malicious. It's really disgusting to need starfish and pearls immediately after you can mint the treasure map. Some people compare the compass, but the starting point of the compass is the same for everyone. The heart bonus should make sense. Some people didn't sell the starfish, they had 20 or more, and I sold them respecting the gameplay.

**Comment by SarielWang93** [comment]
I suspect that the project party specially released the treasure map a few days ago, and then observed how the starfish sold, and then waited for the top 200 and the last one to be less than 10, and then released it impatiently

**Comment by Xeko-92** [comment]
In the docs it says "The Wooden Compass is also quest item that can be used to mint other rare items". I think it's fair to infer from that statement that other items wouldn't be used to mint rare items. If they might be used to mint rare items, why would the docs single out only the wooden compasses?

![image](https://user-images.githubusercontent.com/123136614/220448479-8b40492b-baf1-4b56-a99f-3cefa1c133c5.png)

**Reply by rasheed19871** [reply]
Fair point

**Reply by brian-man** [reply]
> In the docs it says "The Wooden Compass is also quest item that can be used to mint other rare items". I think it's fair to infer from that statement that other items wouldn't be used to mint rare items. If they might be used to mint rare items, why would the docs single out only the wooden compasses?

You have it backwards.  The fact that the compass is called out implies that _literally everything else_ is not worth "saving" for future mints.

**Reply by Xeko-92** [reply]
Interesting that note about wooden compasses was deleted after I posted this...

**Reply by brian-man** [reply]
> Interesting that note about wooden compasses was deleted after I posted this...

TR said something about it on Discord a few hours ago.  Not really related to the post, but the same topic / concern about phrasing.

**Reply by SarielWang93** [reply]
they obviously did it on purpose

**Reply by SarielWang93** [reply]
As a serious player, if there is a slight sign, I will hoard too much, but this backstab is really disgusting

**Comment by datorymen** [comment]
The reason I sold my only Pearl and all star fish was because according to the official document, they are not used for any other purpose, except for sellling at the shop.

Pipi, Wooden Compass, See Weed and Coral etc...these were clearly written as the material for NFT in the document.

So the feeling is that I followed the game rule but suddenly the rule was changed. 

I guess many other players are same as me.  

Is it possible to return us the Pearl and Star fish we sold?   

I think by doing this, we can call it a fair NFT to all players, becaues we all got these items by luck. But the new rule took these luck from us.

**Reply by rasheed19871** [reply]
Is it possible to return us the Pearl and Star fish we sold? if that could happen i will be refunded my pearl and a few starfish

**Reply by datorymen** [reply]
yes, of course, we will refund the SFL. And I think many players are willing to refund 2X or more SFL to get the sold items back...

**Comment by Shumpooo** [comment]
At least, the cost should be compass and iron(or gold).
I keep the compass and sold other treasures, because I thought compass would be next nft's cost(doc also said compass will be the nft's mint cost).
But now this iron idol's mint cost is not good.
I love this game's team and trust them, also the best project I've ever seen.
So I hope team can make a little change with this nft.

**Comment by Romiro11** [comment]
I agree that the +1 iron buff is too much for RNG mechanics.

I have a legacy gold rush and it took a huge amount of time and money to get it, and it gives me only + 0.5 to gold from each vein, and here +1 where, in fact, nothing but luck (or possibly insider information) is required.

Why +1 to iron and why only 200 pieces? what is the general expediency of introducing this NFT now?

Do we have iron deficiency only 200 people? No.

At a minimum, it was necessary to start by balancing the resources on land extensions. This topic has already been raised many times.

We have an imbalance in both the number of iron nodes and the number of stone pickaxes. Here's where to start. Then introduce NFT for mass use with unlimited supply, and only then, having discussed the rules in advance, introduce this iron idol.

I liked this game because it was possible to plan strategy and tactics. Now there are too many RNGs both in the gameplay and in the actions of the developers.

**Reply by Xeko-92** [reply]
Despite this being one of the most upvoted and active threads in the history of the discussion, it looks like we won't get a response from the devs. It's almost universally agreed that this NFT in this state in the way it was launched/handled could've been better, and that should've been obvious from the start. GranTY even mentioned it before the code went live. I'm honestly quite disappointed that the devs haven't reconsidered this NFT, especially since it still hasn't been minted by anyone.

![image](https://user-images.githubusercontent.com/123136614/220718065-e7d46eaf-4a93-42f6-b982-8072cd3d9aa3.png)

**Comment by adamhannigan** [comment]
Thanks for your feedback! ❤️ 

Just a note, the Iron Idol is not intended to address the wider concerns of lack of iron nodes or SFTs. We are addressing this in a range of open proposals at the moment 😄 

This includes:

- Iron Skills - https://github.com/sunflower-land/sunflower-land/discussions/2180
- Iron node balancing - https://github.com/sunflower-land/sunflower-land/discussions/2195
- Iron SFTs - https://github.com/sunflower-land/sunflower-land/discussions/2148

The community discussion around Iron balance is happening on these proposals - we would greatly appreciate your feedback there 🤗

**Reply by Xeko-92** [reply]
Thank you for the response Adam. I don't think the issue is really iron balance, not for me at least, the issue is issuing such a powerful NFT (top 5 ever? Kuebiko, GA, & FB are the only ones that immediately come to mind that are better) in such a random way.

Some people had sold their Starfish before this item was released since they are "bounties" and the compass was the item singled out as a useful item for crafting items (I'm not one of these people luckily for me).

I don't mind powerful items being issued, I think a lot of people just don't like how it was gone about. I think having a larger supply of a less powerful item would be more appropriate for this kind of mint, similar to the heart of davey jones.

**Reply by adamhannigan** [reply]
I hear your frustration.

When we release OP items, we aim that players have an even playing field. An example is no one had wood for the Foreman Beaver or Wheat for the Kuebiko.

We also wanted to ensure that whales were not favoured in this feature. For this reason, we chose the Star Fish since players could not be attained using Sand Drills. This meant that all players had an even playing field and chance of getting this item ❤️

**Reply by Shumpooo** [reply]
Thank you adam.
Now I understand how team will balance the game, and I have no worry about that.

But this iron idol's mint cost still doesn't make sence.
I think it should be made clear from the beginning that all the bounty items would be mint costs. 
I thought compass was the only mint resource in these bounty items(this is also written in docs) . So I didn't sell them. But now starfish and pearl are necessary to mint nft, it's unfair to players who sold all.

I know that team wanted to make every players have chance to mint nft, but the new players and low level bumpkin players have no money at that phase, so they sell the bounties make money(sfl). 
To these players they have no chance to get this nft. And I believe there are many players kind like this.

I hope team can think about this mint cost again. (For example make new bounty item to be the mint cost of iron idol, and make everyone start again in fair situation)

Finally, I want to apologize about my bad English. Hope you can understand.

**Reply by Xeko-92** [reply]
Thanks again for the reply. While I don't prefer this approach (I would prefer if very good items were minted for high prices, for example this could've been like 2000 iron or something), I appreciate the attempt to make things fair to everyone.

**Reply by DominicTing** [reply]
> I hear your frustration.
> 
> When we release OP items, we aim that players have an even playing field. An example is no one had wood for the Foreman Beaver or Wheat for the Kuebiko.
> 
> We also wanted to ensure that whales were not favoured in this feature. For this reason, we chose the Star Fish since players could not be attained using Sand Drills. This meant that all players had an even playing field and chance of getting this item ❤️

Whales play many roles in a game, including stabilizing SFL asset prices, stabilizing SFT asset prices, etc., which can attract more new players. If you suppress whales every time a new SFT is released, this may result in multiple accounts benefiting and losing players who play normally. In my opinion, fairness is relative, and the game should encourage all players to hoard resources instead of dumping them. So are all hoarding resource players whale players?

**Reply by monkeybobby** [reply]
I am very excite adam can answer this issue 
I hope adam can look at my word 

First i want to say , whatever revision rule for mint iron idol 
there always someone feel not fair
i can strike a chord u , u. want to give new guy a good NFT,
u also want to give every player a suprise ,
but there is too sudden,
last for wood , change rule also too sudden, when i just buy 1k wood
i believe have more people lose in wood rule

so i express and i hope 
right now ,  sunflower land already many people a web3 home ,
when a home have a new rule or change 
earlier let more people to know 
the better than to do post a proposal 
let more player can together decide or have a hand in game develop!

Happy Farming!

**Reply by Aquila199** [reply]
> I hear your frustration.
> 
> When we release OP items, we aim that players have an even playing field. An example is no one had wood for the Foreman Beaver or Wheat for the Kuebiko.
> 
> We also wanted to ensure that whales were not favoured in this feature. For this reason, we chose the Star Fish since players could not be attained using Sand Drills. This meant that all players had an even playing field and chance of getting this item ❤️

But the downside was that whales where favoured in away as they where the players that did not have to sell there items as TI is expensive to play without selling at all so i understand the logic but you should have added a new item to TI then it would have been 100% the same luck for all that wanted to play @adamhannigan
